### DISCUSSION TITLE
coordinates - GPS+ altitude- barometer

### DISCUSSION THREAD

**Discussion by ruslan180621** [iNavFlight/inav#10446] [Q&A]
hello, please tell me how you can configure inav so that when the GPS is turned on, the altitude readings are taken from the barometer. the fact is that in our region there is a substitution of coordinates (spoofing) and height. and it is very important that the height does not jump. coordinates - GPS+ altitude- barometer

**Comment by ruslan180621** [comment]
the point is to turn on the GPS altitude

**Comment by sensei-hacker** [comment]
You are looking for the setting inav_w_z_gps_p

https://github.com/iNavFlight/inav/blob/master/docs/Settings.md
